lib/suites.js

```diff
@@ -62,16 +68,28 @@ export function getSuiteParams({config, suiteName}) {
   return {zcap, createSuite, referenceId};
 }
 
-function _createEddsa2022Suite({signer}) {
+function _createEddsaRdfc2022Suite({signer}) {
+  // remove milliseconds precision
+  const date = new Date().toISOString().replace(/\.\d+Z$/, 'Z');
+  const cryptosuite = eddsaRdfc2022CryptoSuite;
+  return new DataIntegrityProof({signer, date, cryptosuite});
+}
+
+function _createEcdsaRdfc2019Suite({signer} = {}) {
   // remove milliseconds precision
   const date = new Date().toISOString().replace(/\.\d+Z$/, 'Z');
-  const cryptosuite = eddsa2022CryptoSuite;
+  const cryptosuite = ecdsaRdfc2019CryptoSuite;
   return new DataIntegrityProof({signer, date, cryptosuite});
 }
 
-function _createEcdsa2019Suite({signer} = {}) {
+function _createEcdsaSd2023Suite({signer} = {}) {
   // remove milliseconds precision
   const date = new Date().toISOString().replace(/\.\d+Z$/, 'Z');
-  const cryptosuite = ecdsa2019CryptoSuite;
+  const cryptosuite = createEcdsaSd2023SignCryptosuite({
+    mandatoryPointers: [
+      '/issuer',
+      '/issuanceDate'
+    ]
+  });
   return new DataIntegrityProof({signer, date, cryptosuite});
```

Comment:
We will also want to set the proof ID:

```
const diProof = new DataIntegrityProof({signer, date, cryptosuite});
diProof.proof = {id: `urn:uuid:${uuid()`};
return diProof;

Reply:
Set [b8a5ff3](https://github.com/digitalbazaar/bedrock-vc-issuer/pull/124/commits/b8a5ff3a150370d5e8a6a37ba58f247382efe34c)